uwblueprint/ccbc
# Get data for admin dashboard

## Notion ticket link

[Admin Dashboard Table](https://www.notion.so/uwblueprintexecs/Admin-Dashboard-Page-8f4bf28e198b4edd802e5b8247075e2a)



## Implementation description
* This ticket will hook up the dashboard table with the actual backend for reviews rather than dummy data. Call create review and get review end-points in backend to do this.



## Steps to test
1. Create/delete reviews and make sure the reviews are reflected on the admin dashboard table



## What should reviewers focus on?
* code correctness


## Checklist
